index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles/styles.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.3.0/css/all.min.css">
+    <title>Salman Ahmad Khan's Portfolio</title>
+</head>
+<body>
+    <header class="container">
+        <nav class="logo">My Logo</nav>
+        <div><i class="fa-sharp fa-solid fa-bars"></i></div>
+      <!-- write these commented elements for later use -->
+        <!--   <div>
+            <ul>
+                <li><a href="#">Menu</a></li>
+                <li><a href="#"></a>Contact Us</li>
+                <li><a href="#"></a>About</li>
+            </ul>
+        </div> -->
+    </header>
+    <main>
+        <section class="container2">
+        <div class="section1">
+        <h1>Hey There. I'm Salman</h1>
+        <h2>I am a full stack developer</h2>
+        <p>I can help you to build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>  
+        </div>
+        <!-- 
+        use section2 for social elements -->
+        <div class="section2">
+            <img class="social"src="/img/Enabled.png" alt="">
+            <img class="social"src="/img/Disabled.png" alt="">
+            <img class="social"src="/img/Disabled-1.png" alt="">
+            <img class="social"src="/img/Disabled-2.png" alt="">
+            <img class="social"src="/img/Disabled-3.png" alt="">
+        </div>
```

Comment:
- [ ] I suggest using the `ul` list tag for the social media links and wrapping each of the `<a>` tags inside `<li>` list tags. Use of appropriate sematic tags is important for the purpose of acccessibility.